test/gtest/uct/ib/test_cqe_zipping.cc

```diff
@@ -150,24 +130,33 @@ struct test_cqe_zipping : public uct_test {
         return ucs_derived_of(p.m_entity->iface(), uct_ib_iface_t)->stats;
     }
 
-    void check_skip_test_tl()
-    {
-        if ((GetParam()->tl_name != "dc_mlx5") &&
-            (GetParam()->tl_name != "rc_mlx5") &&
-            (GetParam()->tl_name != "ud_mlx5")) {
-            UCS_TEST_SKIP_R("not supported yet");
-        }
-
-        check_skip_test();
-    }
+    constexpr static size_t buf_size = 4096;
```

Comment:
Minoir: usually `static constexpr` is used.

Reply:
```suggestion
    static constexpr size_t buf_size = 4096;
```